test/Inputs/clang-importer-sdk/usr/include/ctypes.h

```diff
@@ -224,6 +224,17 @@ typedef struct Dummy {
 
 Dummy * (*getFunctionPointer3(void))(Dummy *);
 
+// These two function types should be serializable despite the struct
+// declarations be incomplete and therefore (currently) unimiportable.
```

Comment:
Typo: "be" -> "being"

Reply:
Thanks, fixed this and the other on this line.